src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -9381,6 +9381,7 @@ public sealed partial class DependencyAttribute : System.Attribute
         public string DependentAssembly { get { throw null; } }
         public System.Runtime.CompilerServices.LoadHint LoadHint { get { throw null; } }
     }
+    [System.ObsoleteAttribute("DisablePrivateReflectionAttribute has no effect in .NET Core and .NET 5.0+ applications.", DiagnosticId = "SYSLIB0013", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
```

Comment:
SYSLIB0013 is already taken.  A new obsoletion ID is needed.

Reply:
It was mentioned in the commentary on this issue, so I decided that it was necessary to use it. Thanks for the explanation. Can I take SYSLIB0015 instead?

Reply:
I hope I did it right :)